tests/dummy/app/views/sparkline-table-cell.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import TableCell from 'ember-table/views/table-cell';
+
+export default TableCell.extend({
+  templateName: 'empty-cell',
+  heightBinding: 'controller.rowHeight',
+
+  onContentOrSizeDidChange: Ember.observer(function() {
+    this.$('svg').remove();
+    return this.renderD3View();
+  }).observes('row', 'width'),
```

Comment:
wrong syntax for an observer ( I'll stop commenting on this one as you can search the occurrences in the repo )